semgrep/semgrep/cli.py

```diff
@@ -118,11 +118,6 @@ def cli() -> None:
         ),
     )
 
-    config.add_argument(
```

Comment:
Instead of removing, can we print out a deprecation warning for now? (I.e. "this will be removed in Semgrep 0.11")

Reply:
What's the significance of `0.11`? The way I've done this in the past is add a deprecation warning for one minor version, then remove in the next minor version.

E.g.

* Current version: `0.4.6`
* Deprecate in version: `0.5.X`
* Remove in version: `0.6.X`

At least until we hit our first major version, then removals require a major version bump.

Reply:
Since this will go out in `0.10`, `0.11` would be the next minor version.

Reply:
We're currently on `0.8.1`, wouldn't this go out in `0.9.0`?

Reply:
Oh I thought `0.9.0` was going out today... cc. @brendongo

Reply:
Fwiw in this case I would be fine just removing?

Reply:
I would be fine with just removing as well. The --exclude-tests upgrade path is super simple and --r2c will almost certainly have zero users.

Reply:
let's do it